### DISCUSSION TITLE
Setting border color to an extend custom color, even in dark mode

### DISCUSSION THREAD

**Discussion by jsoneaday** [tailwindlabs/tailwindcss#7200] [Help] [answered]
I have created several custom colors inside of tailwind.config.js. This was done in the theme section as shown below. I have a color for light mode and a color for dark mode. I am using the class darkMode setting in order to switch between the two automatically. I'm trying to set my border color but it's not working. Here's my try below. I also tried border-[fg-secondary dark:fg-dk-secondary]

```
<div className="mb-4 border-bottom border-solid border-fg-secondary-900 dark:border-fg-dk-secondary-900">
      <img src={logo} alt="logo" style={{ width: "40px" }} />
</div>
```

```
module.exports = {
  content: ["./src/**/*.{js,jsx,ts,tsx}"],
  darkMode: "class",
  theme: {
    extend: {
      colors: {
        "fg-primary": "black",
        "fg-dk-primary": "white",
        "fg-secondary": "#121212",
        "fg-dk-secondary": "white",
        tertiary: "#808080",
        primary: "white",
        "dk-primary": "black",
      },
    },
  },
  plugins: [],
};
```

**Comment by macbruker** [comment, marked as answer]
I think`border-bottom` should be `border-b`. Does that help?

**Comment by jsoneaday** [comment]
Oh no that did work. Sorry and thanks!
